### PR TITLE
WebAPI and Minimal API example for .NET 6

### PR DESCRIPTION
**Pull request by xiaomi7732**
Part of #265. This is for the WebAPI & Minimal Web API.

### CODE REVIEW

**Review by karolz-ms — APPROVED** [review]
Looks great

File: examples/WebAPI.Net6/MinimalAPI/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "applicationUrl": "https://localhost:7224;http://localhost:5015",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "IIS Express": {
```
[the remark this answers is not in the document]
Comment (xiaomi7732) [reply examples/WebAPI.Net6/MinimalAPI/Properties/launchSettings.json:22]: Don't know. Does anybody have an answer?

File: examples/WebAPI.Net6/Readme.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        6.0.302
+        ```
+    _You might have a slightly different patch version and that should be fine._
+
+* A Kubernetes Cluster that you can manage with `kubectl`.
+    * If you don't have any, there are several options:
+        * [Azure AKS](https://docs.microsoft.com/en-us/azure/aks/)
+        * [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+        * [MiniKube](https://minikube.sigs.k8s.io/docs/start/)
+        * ...
```
Comment (JacobBovee) [inline examples/WebAPI.Net6/Readme.md:22]: Don't know if you meant leave this "..." here

**Review by JacobBovee — COMMENTED** [review]
LGTM. Just noticed one small typo.

File: examples/WebAPI.Net6/Readme.md (hunk 2 above)
Comment (xiaomi7732) [reply examples/WebAPI.Net6/Readme.md:22]: Thanks for the question. Let me double check.

Comment (xiaomi7732) [reply examples/WebAPI.Net6/Readme.md:22]: There is actually more than just what I listed. I want to express that it is not an exhausted list. What would you put there?

**Comment by xiaomi7732** [comment]
> LGTM. Just noticed one small typo.

Thanks @JacobBovee . Just a double check, by typo, do you mean the `...` you pointed out, or are you referring somewhere else? Mind to point it out?

**Comment by JacobBovee** [comment]
> > LGTM. Just noticed one small typo.
> 
> Thanks @JacobBovee . Just a double check, by typo, do you mean the `...` you pointed out, or are you referring somewhere else? Mind to point it out?

Oh yeah it's the "..." no problem if you just meant it as etc. I think that makes sense.

**Review by JacobBovee — APPROVED** [review]

File: examples/WebAPI.Net6/Readme.md (hunk 2 above)
Comment (xiaomi7732) [reply examples/WebAPI.Net6/Readme.md:22]: Cool. I'll leave it as is based on our conversation below. Hope that won't be confusing. And I guess we could update it easily.

Comment (JacobBovee) [reply examples/WebAPI.Net6/Readme.md:22]: That looks good. Definitely a fair use of the ellipses.

**Pull request merged**